client/js/views/Rules.vue

```diff
@@ -123,72 +152,79 @@
       <v-row class="d-flex flex-column mb-4">
         <h1 class="gradient-text">Multiplayer Variants</h1>
         <p class="mb-4">
-          Presently only 2-player cuttle is available at
-          <a href="https://www.cuttle.cards"> www.cuttle.cards</a>, however Cuttle can be played
-          with 2-4 players. 3 and 4-player Cuttle each have slight variations on the rules to
-          optimize the game balance while changing the pace and the feel. Give them a try next time
-          you're playing Cuttle in a group!
+          Presently only two-player Cuttle is available at
+          <a href="https://www.cuttle.cards"> www.cuttle.cards</a>; however,
+          Cuttle can be played with 2-4 players. Three and four-player Cuttle
+          each have slight variations on the rules to optimize the game balance
+          while changing the pace and the feel. Give them a try next time you're
+          playing Cuttle in a group!
         </p>
         <h3>3 Player Cuttle: Cutthroat</h3>
         <p class="mb-4">
-          Cutthroat is a brutally fast-paced 1v1v1 version of Cuttle, full of betrayal and pivotal
-          moments!
+          Cutthroat is a brutally fast-paced 1v1v1 version of Cuttle, full of
+          betrayal and pivotal moments!
         </p>
         <p>The rules are the same as above with the following changes:</p>
         <ul>
           <li>
-            The win condition is reduced. Players need 14 points to win by default, 9 points with
-            one king in play, 5 points with two kings in play, and having 3 kings in play
-            immediately wins (goal becomes zero).
+            The win condition is reduced. Players need 14 points to win by
+            default, 9 points with one king in play, 5 points with two kings in
+            play, and having 3 kings in play immediately wins (goal becomes
+            zero).
           </li>
           <li>
-            To start play, deal each player 5 cards and the player to the left of the dealer goes
-            first. Which player is the delete rotates clockwise each game.
+            To start play, deal each player 5 cards and the player to the left
+            of the dealer goes first. Which player is the delete rotates
```

Comment:
You'll probably want to sync your fork since I fixed this typo (among others) in #160